src/TextForEvent.js

```diff
@@ -248,6 +248,26 @@ function textForPowerEvent(event) {
     });
 }
 
+function textForWidgetEvent(event) {
+    const senderName = event.sender ? event.sender.name : event.getSender();
+    const previousContent = event.getPrevContent() ? event.getPrevContent() : {};
+    const {name, type, url} = event.getContent() ? event.getContent() : {};
+    let widgetName = widgetName || name || type || previousContent.type || '';
+
+    // Apply sentence case
+    widgetName = widgetName ? widgetName[0].toUpperCase() + widgetName.slice(1).toLowerCase() + ' ' : '';
```

Comment:
Is there any good reason to lower-case the rest of the string? I think maybe preserving the case of the rest of the string is better. That is, just upper-case the first character and leave the rest as it is.